contracts/ocr2/src/contract.rs

```diff
@@ -73,6 +73,9 @@ pub fn instantiate(
         billing: Billing {
             recommended_gas_price: 0,
             observation_payment: 0,
+            base_gas: None,
+            gas_per_signature: None,
+            gas_adjustment: None,
```

Comment:
These are now settable via `setBilling` but can also be `nil` to use the built-in default